hphp/hack/hhi/stdlib/builtins_class.hhi

```diff
@@ -12,17 +12,17 @@ function get_declared_classes();
 function get_declared_interfaces();
 function get_declared_traits();
 function enum_exists(string $class_name, bool $autoload = true): bool;
-function class_exists($class_name, bool $autoload = true): bool;
-function interface_exists($interface_name, bool $autoload = true): bool;
-function trait_exists($trait_name, bool $autoload = true): bool;
-function get_class_methods($class_or_object);
-function get_class_vars($class_name);
+function class_exists(string $class_name, bool $autoload = true): bool;
```

Comment:
This is the hhi, not the hni, so classname should work. Might be overly-strict though.

cc @jwatzman